embed/src/app.py

```diff
@@ -48,18 +51,21 @@ def embed_queue_item(self, queue_item: EmbedQueueItem):
         """
 
         chunk: Chunk = self.storage_handler.read_item(queue_item.chunk_uuid, "Chunk")
-        embedded_sentences = embed_sentences(queue_item.model, [chunk.text])
+        embedded_sentences = model_db.embed_sentences(queue_item.model, [chunk.text])
         if len(embedded_sentences.data) != 1:
-            raise ValueError(f"expected just 1 embedding but got {len(embedded_sentences.data)}")
+            logging.error(f"expected just 1 embedding but got {len(embedded_sentences.data)}")
+            return
         chunk.embedding = embedded_sentences.data[0].embedding
-        self.storage_handler.update_item(queue_item.chunk_uuid, chunk)
+        self.storage_handler.update_item(chunk.uuid, chunk)
+        logging.info(f"embedded: {chunk.text}")
```

Comment:
How big could these chunks be, are we going to flood the logs here? Would it be useful to have the uuid of the chunk instead if that's the case?

Reply:
they should be small... but you are right, this was just for local debugging tbh. will remove